Add issue templates (#12521)
vaadin/flow: .github/ISSUE_TEMPLATE/bug_report.md

---
name: Bug report
about: Issues for Flow related problems
title: ''

> labels: ''

Reviewer: No `bug` label?

Author: It's not a bug until triaged.

assignees: ''

---

<!--
Please READ these instructions & USE the issue template below, Thank You!

For general support from the community, use https://stackoverflow.com/questions/tagged/vaadin (or tag vaadin-flow) or Vaadin discord chat https://discord.gg/MYFq5RTbBn instead.

> NOTE: Issues concerning UI components (e.g. Button, Grid, TextField, ...) should go to the components repository https://github.com/vaadin/flow-components .

Reviewer: Another note: Issues concerning Vaadin Fusion should go to the Fusion repository https://github.com/vaadin/fusion

Author: done

---
name: Bug report
about: Issues for Flow related problems
title: ''
labels: ''
assignees: ''

---

<!--
Please READ these instructions & USE the issue template below, Thank You!

> Only use GitHub issues for bugs and feature requests.   

Reviewer 2: ```suggestion

```

For general support from the community, use https://stackoverflow.com/questions/tagged/vaadin (or tag vaadin-flow) or Vaadin discord chat https://discord.gg/MYFq5RTbBn instead.

> NOTE: Issues concerning certain UI components should go to the components repository https://github.com/vaadin/flow-components .

Reviewer 2: ```suggestion
NOTE: Issues concerning UI components (e.g. Button, Grid, TextField, ...) should go to the components repository https://github.com/vaadin/flow-components .
```

> For feature requests, always include your use case - what are you trying to achieve and why.

Reviewer 2: ```suggestion
```

Good quality bug report increases the likelihood to get the bug fixed. A bad quality one will likely be just closed. Please use the following template to report bugs.
-->

> ### Description of the bug / feature

Reviewer 2: ```suggestion
### Description of the bug
```

> ### Description of the bug / feature

Reviewer 2: ```suggestion
### Description of the bug
```

> <!-- Explain briefly what is broken or what you want to achieve -->

Reviewer 2: ```suggestion
<!-- Explain briefly what is broken -->
```
